tests/ComTests.cpp

```diff
@@ -614,6 +617,7 @@ interface __declspec(uuid("ececcc6a-5193-4d14-b38e-ed1460c20a01"))
 IDerivedTest : public ITest
 {
    STDMETHOD_(void, TestDerived)() = 0;
+   virtual ~ITest() = default;

```

Comment:
This should be `~IDerivedTest` - you can't declare a destructor for a base type.